.github/scripts/build_base_docker.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
```

Comment:
looks like we should use INCULDE from this extension
https://codeberg.org/devthefuture/dockerfile-x

Dockerfile.base with all the stuff related to refact docker (vllm should not be included!)
then for build we use simple `docker build .` command and for github build system we can replace INCLUDE with FROM or make another Dockerfile for it

this approach will be better due to less code and more clear working pipeline

and finally I request you to write a README because it's unclear how to use this 2 bash scripts